src/run.js

```diff
@@ -193,7 +193,12 @@ const minimalcss = async options => {
         })
         .forEach(stylesheet => {
           if (!stylesheetAstObjects[stylesheet.href]) {
-            throw new Error(`${stylesheet.href} not in stylesheetAstObjects!`)
+            if (stylesheet.href.startsWith('blob:')) {
+              // TODO: do we need to do something here?
```

Comment:
I think this is the right thing to do. It's sad but it's what it is. 

It's not that different from the fact that `minimalcss` doesn't clean up any `<style>` tags on the page. (Perhaps it should in a future version. But that's a story for another day) 
If someone uses a blob to inject a bunch of "fat" CSS then I think it's best to just ignore it. 

However, I would like one change. Can you remove this if statement and weave it into the filter loop a couple of lines above?
Perhaps something like this:
```javascript
.filter(link => {
    return (
       link.href && !link.href.startsWith('blob:') &&
       ...
    )
})
```
What do you think about that?